### PR TITLE
fixed code quality issues for address_dialog python file

### PR DESCRIPTION
**Pull request by coolguy-cell**
flake8 fixes
pycodestyle fixes
pylint fixes

### CODE REVIEW

File: src/bitmessageqt/address_dialogs.py [hunk 1]
```diff
@@ -105,7 +125,7 @@ class AddAddressDialog(AddressDataDialog):
 
     def __init__(self, parent=None, address=None):
         super(AddAddressDialog, self).__init__(parent)
-        widgets.load('addaddressdialog.ui', self)
+        widgets.load("addaddressdialog.ui", self)
```
Comment (g1itch) [inline src/bitmessageqt/address_dialogs.py:128]: What's wrong with single quote?

File: src/bitmessageqt/address_dialogs.py [hunk 2]
```diff
@@ -32,50 +39,58 @@ def addressChanged(self, QString):
...
+                    "The address is not typed or copied correctly"
+                    " (the checksum failed)."
+                )
+            )
+        elif status == "versiontoohigh":
+            self.labelAddressCheck.setText(
+                _translate(
+                    "MainWindow",
+                    "The version number of this address is higher than this"
+                    "software can support. Please upgrade Bitmessage."
```
Comment (g1itch) [inline src/bitmessageqt/address_dialogs.py:69]: missing space

**Comment by g1itch** [comment]
I don't understand what issues did you fix in `bitmessageqt.address_dialogs`? It passed all checks many times with no warnings.

**Comment by PeterSurda** [comment]
@g1itch no worries, this is my fault, I wasn't paying enough attention to the results of the automated checks and gave wrong feedback.

**Comment by g1itch** [comment]
> @g1itch no worries, this is my fault, I wasn't paying enough attention to the results of the automated checks and gave wrong feedback.

So, you prefer the formatting like this?
```python
            queues.UISignalQueue.put(
                (
                    "updateStatusBar",
                    _translate(
                        "EmailGatewayDialog",
                        "Sending email gateway registration request"
                    )
                )
            )
```

Can you please tell me why it's better?

**Comment by PeterSurda** [comment]
> Can you please tell me why it's better?

I don't care, do whichever you prefer. As I said, I gave @coolguy-cell incorrect feedback. I saw that the automated checks showed no improvements but I thought the checks are faulty. The current automated checks are a dirty hack that often breaks and I don't have time to fix them. I just want to get the CQ issues to 0 so the move to buildbot's checks is easier. If a hunk doesn't affect the CQ results it should be rejected like you said.

**Pull request closed without merging**